hcloud/volume.go

```diff
@@ -188,6 +193,13 @@ func (c *VolumeClient) Create(ctx context.Context, opts VolumeCreateOpts) (Volum
 			reqBody.Location = opts.Location.Name
 		}
 	}
+	if opts.Automount != nil {
```

Comment:
Condition not needed. You can set `Automount` directly when constructing `reqBody`.